pkg/config/setup/config.go

```diff
@@ -1286,6 +1286,7 @@ func telemetry(config pkgconfigmodel.Setup) {
 
 	// Agent Telemetry
 	config.BindEnvAndSetDefault("agent_telemetry.enabled", true)
+	config.BindEnvAndSetDefault("agent_telemetry.compression_level", 1)
```

Comment:
Could you also declare the `agent_telemetry.use_compression` config please ? (added in the same PR mentioned in the description)

Reply:
@pgimalac it is done